x/staking/simulation/operations.go

```diff
@@ -120,12 +120,15 @@ func SimulateMsgCreateValidator(
 		address := sdk.ValAddress(simAccount.Address)
 
 		// ensure the validator doesn't exist already
-		_, found := k.GetValidator(ctx, address)
-		if found {
+		_, err := k.GetValidator(ctx, address)
+		if err == nil {
```

Comment:
This should be updated for checking the error type.

Reply:
I believe this is fine, having no error means it was found